### PR TITLE
Added support for the 'expm1' op and its VJP.

### PR DESCRIPTION
**Pull request by eaplatanios**

### CODE REVIEW

**Comment by eaplatanios** [comment]
Added a test similar to #144.

**Comment by eaplatanios** [comment]
@rxwei this should be ready now.

File: Sources/TensorFlow/Operators/Math.swift [hunk 1]
```diff
@@ -1537,7 +1542,6 @@ internal func _vjpMatmul<Scalar: TensorFlowFloatingPoint>(
     transposed transposeB: Bool = false
 ) -> (Tensor<Scalar>, (Tensor<Scalar>) -> (Tensor<Scalar>, Tensor<Scalar>)) {
     let value = matmul(lhs, transposed: transposeA, rhs, transposed: transposeB)
-    return (value, { v in
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]: Looks like an unintended deletion.

File: Sources/TensorFlow/Operators/Math.swift [hunk 2]
```diff
@@ -1517,16 +1532,6 @@ public func matmul<Scalar: Numeric>(
     _ rhs: Tensor<Scalar>,
     transposed transposeB: Bool = false
 ) -> Tensor<Scalar> {
-    switch (lhs.rank, rhs.rank) {
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]: Here too.

**Comment by eaplatanios** [comment]
I'll need to go now, but will continue with this tomorrow morning. This along with #151, #156, and #159 should be ready to test now.

**Comment by rxwei** [comment]
All tests passed.

**Pull request merged**